src/main/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerIssueAos.java

```diff
@@ -57,20 +75,66 @@ public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuild
                 CASEWORKER_LEGAL_ADVISOR));
     }
 
+    public AboutToStartOrSubmitResponse<CaseData, State> aboutToSubmit(final CaseDetails<CaseData, State> details,
+                                                                       final CaseDetails<CaseData, State> beforeDetails) {
+
+        final CaseData caseData = details.getData();
+        final Long caseId = details.getId();
+        log.info("Caseworker issue AOS about to submit callback invoked. Case ID: {}", caseId);
+
+        if (!caseData.getApplication().isPersonalServiceMethod()) {
+
+            final Applicant respondent = caseData.getApplicant2();
+            final Solicitor respondentSolicitor = respondent.getSolicitor();
+
+            if (respondent.isRepresented()) {
+
+                if (respondentSolicitor.hasDigitalDetails()) {
+                    log.info("Sending respondent AOS pack to bulk print, "
+                        + "respondent is represented by digital solicitor.  Case ID: {}:", caseId);
+                    aosPackPrinter.print(caseData, caseId);
+                    setNoticeOfProceedingsInformation(caseData, respondentSolicitor);
+                } else {
+                    log.info("Not sending respondent AOS pack to bulk print, "
```

Comment:
We may be able to take this case out - I was told that all solicitors will be digital by the time we go live

Reply:
I have removed the test for respondent digital solicitor here and for the notifications
https://github.com/hmcts/nfdiv-case-api/pull/429/commits/137403c084fe0a27d56dbda3ef9f97bd4473b5de